CONTRIBUTING.md

```diff
@@ -0,0 +1,64 @@
+# Contributing to Pride in London app
+
+If you're interested in contributing code, be it a simple bug fix or a feature
+to the Pride app, this document should hopefully be helpful to get your
+changes in as smoothly as possible.
+
+## Contributing code
+
+### Pull requests
+
+The Pride app follows a standard [Github flow](https://guides.github.com/introduction/flow/).
+Start by forking the repo and starting a branch off of `master`. We do what
+we can to keep the `master` branch working using automated testing and
+continuous integration ([CircleCI](https://circleci.com/)).
+
+Pick a short but descriptive brach name. We encourage you to open a Pull Request
+early, you can mark it as work in progress by adding '[WIP]' to the title. This
+helps us to provide feedback on your implementation early.
+
+Try to make pull requests as small as they can be, it makes them easier to
+review and quicker to integrate. It's ok to split a single feature across
+multiple PRs as long as the app works fine and the UX isn't negatively
+impacted after each pull request.
+
+Try and provide a thorough description of what you've done and some reasoning
+behind it, so that reviews can be done quickly and minimal back and forth is
+required.
+
+### Code style
+
+Code style is checked and enforced by [Prettier](https://prettier.io/), we
+highly recommend installing an extension for your favourite editor and
+formatting on save.
+
+Basic static analysis is done by [ESLint]()https://eslint.org/, most editor
+support error highlighting, which is also highly recommended.
+
+### Unit tests and test coverage
+
+Pride app uses [Jest](https://facebook.github.io/jest/) as a test runner. Run
+
+```
+$ yarn test
+```
+
+to run unit tests. When working on a feature it may be useful to run Jest in
+watch mode by
+
+```
+$ yarn test -- --watch
```

Comment:
If you have the latest version of yarn, it will complain about the `--`. It suggest using `yarn test --watch` here.